website_product_filters/static/src/js/test_attributes.js

```diff
@@ -24,11 +24,10 @@
             );
         }
         if (steps[k].title === "open customize menu bis") {
-            steps.splice(k, 1, {
-                title: 'Open Customize Menu',
-                element: '#customize-menu-button',
-            }
-            );
+            steps.splice(k, 1);
+        }
+        if (steps[k].title === "click on 'Product Attribute's Filters'") {
+            steps.splice(k, 1);
```

Comment:
PLEASE PLEASE PLEASE 

NO IF IN TESTS - NO IF IN TESTS - NO IF IN TEST
---

If there is onea document the rationale explaining properly why are you braking such rule.